contracts/scripts/v0.1/utils.cjs

```diff
@@ -215,6 +215,38 @@ function getFormattedDate() {
   return moment().format('YYYYMMDDHHMMSS')
 }
 
+// returns object with {key(contractName):value(address)} inside dirPath
+async function loadDeployments(dirPath) {
```

Comment:
It will be really good if we have tests for these kind of functions because it includes a lot of domain knowledge and reasoning that is not obvious.

Reply:
BTW we already have similar function. How about reusing that one?

https://github.com/Bisonai/orakl/blob/9433d538c7faae272308a256559bd40ced7115b6/contracts/utils/index.ts#L21-L61

Reply:
I agree with test codes for complicated and unobvious function, but it seems there's no specific test code which is already written for the script or utility codes. I'll create an issue to be solved in further tasks.

And about using ts file as you've mentioned, importing ts file from cjs file isn't possible. Technically, it's possible but not compatible leading to multiple errors. Or built js file in `dist` can be used, however calling `yarn build` will be required before deploying the contract.

Reply:
Right! The issue with hardhat bites us here again 😢